.github/renovate.json

```diff
@@ -0,0 +1,60 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base",
+    ":prHourlyLimit4",
+    ":prConcurrentLimit20",
+    ":pinAllExceptPeerDependencies",
+    ":semanticCommits",
+    ":semanticCommitScope(deps)",
+    "docker:disableMajor",
```

Comment:
I think we want to get notified about major updates as well. Especially with NGINX, there's typically not much that can go wrong.

Reply:
fair, this on is another of my "copy-pasta" config, where I use this to prevent getting notified of NodeJS 18 (Current) where I want to stay on 16 (LTS).
Yet there might be a better solution to this I have usually opted to use this preset over that till now.

Anyway, I shall remove that preset from the list, as it is more a hindrance here.